src/runtime/pipeline/pipeline_struct.h

```diff
@@ -25,20 +25,17 @@
 #include <limits>
 #include <string>
 #include <unordered_map>
+#include <utility>
 #include <vector>
+#define GLOBAL_MODULE_INDEX -1
```

Comment:
Please remove the macro here, since it has been used only once.

Reply:
fixed.